httpbin/README.md

```diff
@@ -0,0 +1,57 @@
+LocalStack httpbin extension
+===============================
+
+A simple HTTP Request & Response Service directly in LocalStack
+using [httpbin](https://github.com/postmanlabs/httpbin).
+Get the full httpbin experience directly in LocalStack without connecting to httpbin.org!
+
+The httpbin API is served through the hostname `http://httpbin.localhost.localstack.cloud:4566`.
+
+## Install from GitHub repository
+
+Install the extension directly from the GitHub repository by running:
+
+```bash
+localstack extensions install "git+https://github.com/localstack/localstack-httpbin-extension/#egg=localstack-httpbin-extension&subdirectory=httpbin"
```

Comment:
link seems to be wrong

should be 

`"git+https://github.com/localstack/localstack-extensions/#egg=localstack-httpbin-extension&subdirectory=httpbin"`

afaict

Reply:
actually ideally it's just `localstack-extension-httpbin` once it's released. changed it :-)